package/base-files/files/bin/config_generate

```diff
@@ -239,6 +239,7 @@ generate_static_system() {
 		set system.@system[-1].ttylogin='0'
 		set system.@system[-1].log_size='64'
 		set system.@system[-1].urandom_seed='0'
+		set system.@system[-1].require_image_signature='0'
```

Comment:
Why not '1'?

Reply:
This would confuse everyone as adding their own images would *fail* to install. Like a soft vendor lock.

Reply:
aka introduce this feature somewhat slowly

Reply:
If you flip this on by default then the error message when the check fails should tell people how to disable the check or flash anyway.

Reply:
Don't take me wrong, I'm all in to enable it by default as it's going to improve a security noticeably.

* we should probably find a way how to enable it by default for development snapshots first and once tested we could enable it for releases as well

  * once this PR is in, feel free to propose the patch(through mailing list please for wider audience) which would enable it in ath79 platform first, which is still in development phase so it would probably cause less friction

* before we could enable it in releases, I think, that we would need to add support for this in LuCI (possible error handling, modification of this setting), in other words assure, that we deliver good UX for this feature

Reply:
I'd surely keep this off for now, at least until all targets supported metadata including signatures.

I'm happy to work on the UX as well once this here is merged. Would it make sense to add extra error codes for "failed signatures" so sysupgrades via ubus can have a more verbose output?